playbooks/templates/rax-maas/ceph_rgw_stats.yaml.j2

```diff
@@ -9,11 +9,14 @@ type        : agent.plugin
 label       : "{{ check_name }}"
 period      : "{{ maas_check_period_override[label] | default(maas_check_period) }}"
 timeout     : "{{ maas_check_timeout_override[label] | default(maas_check_timeout) }}"
-disabled    : "{{ (check_name | match(maas_excluded_checks_regex)) | ternary('true', 'false') }}"
+disabled    : "{{ (inventory_hostname != groups['rgws'][0] or check_name | match(maas_excluded_checks_regex)) | ternary('true', 'false') }}"
```

Comment:
@tonytan4ever can you rebase? Looks like this was reverted: https://github.com/rcbops/rpc-maas/pull/502